endoreg_db/services/video_import.py

```
@@ -158,181 +273,566 @@ def _create_or_retrieve_video_instance(self):
+            self.processing_context['frames_extracted'] = False
         
         # Ensure default patient data
-        self._ensure_default_patient_data()
+        self._ensure_default_patient_data(video_instance=video)
         
         self.logger.info("Processing environment setup completed")
 
     def _process_frames_and_metadata(self):
         """Process frames and extract metadata with anonymization."""
         # Check frame cleaning availability
         frame_cleaning_available, FrameCleaner, ReportReader = self._ensure_frame_cleaning_available()
```

Ruff (0.14.0)

Unpacked variable `ReportReader` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)